src/main/java/core/basesyntax/RandomGeometricValue.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class RandomGeometricValue {
+    public static final int LOWER_BOUND = 1;
+    public static final int UPPER_BOUND = 46340;
+
+    private Random random = new Random();

```

Comment:
The Random object could be made final since it is not expected to change after being initialized.